workers/main/src/common/utils.test.ts

```diff
@@ -0,0 +1,58 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
```

Comment:
Why file name 'utils.test.ts', I don't see any util in this folder

Reply:
We actually have a file named `utils.ts` in this folder, which contains shared utility functions (`validateEnv`). 
The test file `utils.test.ts` is intended to cover the logic in `utils.ts`.